src/main/java/io/wisetime/connector/ServerRunner.java

```diff
@@ -86,6 +86,9 @@ public WebAppContext getWebAppContext() {
   }
 
   public void startServer() throws Exception {
+    // Ensure that WiseTimeConnector is initialized
+    startServer(true);
+
     final TagRunner tagRunTask = new TagRunner(wiseTimeConnector::performTagUpdate);
```

Comment:
Calling wiseTimeConnector::performTagUpdate without initializing the connector will throw NPE